src/js/components/SortPicker.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react'
+
+const getText = (groupByRow) => {
+  if(groupByRow) return 'Group by Edit'
+  return 'Group by Row'
+}
+
+const getClass = (groupByRow) => {
```

Comment:
Any reason to change the color of the button in this case? I don't think the button should change other than the text. If we want to make this more apparent we could have both options present all the time, highlighting the current selection with the [`active` state](https://standards.usa.gov/buttons/).